mkinitfs.in

```diff
@@ -103,7 +103,7 @@ find_kmods() {
 }
 
 initfs_kmods() {
-	[ -z "$nokernel" ] || return
+	[ "$nokernel" ] && return 0
```

Comment:
This may halt the script on some shell implementations when running with `set -e` and the condition is evaluated as false. That's why reverse logic is used.

Beside that, it should behave exactly the same as before, so I don't understand your change.

Reply:
>This may halt the script on some shell implementations when running with set -e

This is just horrifying. Shouldn't this problem not be delegated to those implementations instead of making our lifes harder? But oh well I partly revert it.

>and the condition is evaluated as false.

Which is why the whole script stops at  https://github.com/alpinelinux/mkinitfs/blob/master/mkinitfs.in#L261